src/configuration.ts

```diff
@@ -13,13 +13,6 @@ function getGitpodHost() {
     return vscode.workspace.getConfiguration('gitpod').get<string>('host', 'https://gitpod.io/');
 }
 
-function getUseLocalApp(useLocalSSHServer?: boolean) {
-    if (useLocalSSHServer) {
-        return false;
-    }
-    return vscode.workspace.getConfiguration('gitpod').get<boolean>('remote.useLocalApp', false);
```

Comment:
Revert this except, so all flags are fetched the same way, wxcept useLocalSSHServer the param we can remove that